ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -73,9 +59,9 @@ tests:
       READY_WAIT_TIMEOUT: 30m
       REPLICAS: "24"
     test:
-    - chain: openshift-qe-node-density
+    - ref: openshift-qe-cluster-density-v2
```

Comment:
to keep consistency do we want to use the `chain: openshift-qe-ocp-qe-perfscale-ci-tests` here instead?

Reply:
Let us do it as part of https://github.com/openshift/release/pull/42602 for consistency, unless we have a dependency here.

Reply:
Will keep this PR to update the node-count, will tackle the ROSA changes seperately.